bigquery/google/cloud/bigquery/job.py

```diff
@@ -1278,6 +1284,20 @@ def from_api_repr(cls, resource, client):
         job._set_properties(resource)
         return job
 
+    @property
+    def query_plan(self):
```

Comment:
I think we should mark as beta until we do have a class to represent these. (Mostly for type conversions of strings in JSON to ints)